README.md

```diff
@@ -127,6 +127,44 @@ docker cp ${CONTAINER}:/root/BUILD/rpms/x86_64/ ./rpms/
 docker rm ${CONTAINER}
 ```
 
+## Building a Hotfix
+
+Sometimes it is necessary to apply a patch without rebuilding the whole release in order to
+deliver a fix quickly.  This can be accomplished by building a hotfix rpm which will apply
+a series of patches.
+
+Create a patch file using `git format-patch` and copy to `rpm_spec/patches/`.  You will have to modify the path to be relative to the RPM BUILDDIR.
+
+For example if we wanted to create a hotfix to apply a commit in radjabov `1286aa34992340cad7a0778df01a845f1772e561` we would run:
+```sh
+git format-patch 1286aa34992340cad7a0778df01a845f1772e561^..1286aa34992340cad7a0778df01a845f1772e561
+```
+
+Which will create a patch file: `0001-Merge-pull-request-23123-from-agrare-fix_miq_request.patch`
+
+Next we have to change the file location to match the RPM BUILDDIR, for a patch in the core rpm this will be e.g. `manageiq-core-18.0-1`
```

Comment:
I'm not totally sure what you did here.

Can you show us a diff of the `fix_miq_request.patch`?
Was it just a change to first line?

Yes, meta, a diff of a diff, but just trying to understand what was done here.
or maybe show it as a `sed` command?

Reply:
We also need to mention that you should remove any spec files from the patches.